Feature/md 469

cloudkserg opened the pull request:
https://chronobank.atlassian.net/projects/MD/issues/MD-468?filter=myopenissues
added SecuriteSignature.save rabbitMq to middleware events
added routes:
get /me/addresses - get addresses from signatureModel
post /signin/signature/addresses - save addresses in signatureModel

Commit:
https://chronobank.atlassian.net/projects/MD/issues/MD-469 - Laborx backend - added send rabbitmq message about token created and test for it

Commit:
https://chronobank.atlassian.net/browse/MD-469 - https://chronobank.atlassian.net/browse/MD-468 - middleware auth flow - added SecuritySignature.js rabbitmq and AddressSignatureModel for user with route = get /me/addresses and post signin/signature/addresses

Commit:
Merge branch 'develop' into feature/MD-469

ipavlenko commented on lib/utils/lib/keystoneUtils.js:
```diff
@@ -5,7 +5,7 @@ const tmp = require('tmp-promise')
 const { promisify } = require('util')
 
-const uploadCloundinaryImage = (...args) => promisify(cloudinary.v2.uploader.upload)(...args)
+const uploadCloundinaryImage = () => {} // (...args) => promisify(cloudinary.v2.uploader.upload)(...args)
```
WTF?

ipavlenko commented on lib/app.js:
```diff
@@ -61,6 +67,8 @@ function createApp (options) {
   }
 
   keystone.startAsync = async function () {
+  await sleep(10000);
```
Please remove

ipavlenko commented on lib/rabbit/lib/InternalMiddlewareProducer.js:
```diff
@@ -0,0 +1,26 @@
+const AbstractFanoutProducer = require('../AbstractFanoutProducer')
+
+class InternalMiddlewareProducer extends AbstractFanoutProducer {
```
You do not need extra Producer here.
You can use general profileProducer, please generalize it.
Use routingKey (you use 'user.created' string for your purposes - it is ok)

If we will create a lot of queues we get some kind of "channel hell"

ipavlenko commented on lib/keystone/SecuritySignature.js:
```diff
@@ -17,6 +17,9 @@ SecuritySignature.add({
   type: {
     type: Types.Select,
     options: [
+      { value: 'bitcoin-address', label: 'Bitcoin Address' },
```
bitcoin gold, bitcoin cash, litecoin, etc.

Commit:
Update SecurityService for reduce addresses to more float structure

Question about it:
Need i use this code or refactor part in middleware with work addreses as {type, value}?
